src/plugins/cosmos/components/StakedRow/StakedRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { Flex, FlexProps } from '@chakra-ui/layout'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { Amount } from 'components/Amount/Amount'
+import { Text } from 'components/Text'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type StakedRowProps = {
+  assetSymbol: string
+  cryptoStakedAmount: BigNumber
+  fiatRate: BigNumber
+  apr: BigNumber
+}
+export const StakedRow = ({
+  cryptoStakedAmount,
+  fiatRate,
+  assetSymbol,
+  apr,
+  ...styleProps
+}: StakedRowProps & FlexProps) => (
+  <Flex width='100%' justifyContent='space-between' {...styleProps}>
+    <Flex height='20px'>
```

Comment:
> APR*

`<AprTag />` was initially added and used in `<StakingOpportunitiesRow />` (https://github.com/shapeshift/web/pull/1001), but the rationale was to have it available for future components that would need iit.
IMO, it's fine to keep it as is for consistency but I'll double check with product! Either way, changing it here would also change it for `<StakingOpportunitiesRow />` (unless we make it so that `APR*` text is optionally appended depending on a prop), so I'd make it a separate commit.

Reply:
I'll remove it from here but it has to come back in `AprTag />` as `...styleProps` instead. We need it to give the tag some proper height according to design.